.travis.yml

```diff
@@ -1,18 +1,18 @@
 language: node_js
 node_js:
-- 8.10.0
+  - 8.10.0
 cache:
   directories:
-  - "$HOME/.cache"
+    - "$HOME/.cache"
 install:
-- yarn install --frozen-lockfile
-- yarn cy:verify
-- yarn build
+  - yarn install --frozen-lockfile
+  - yarn cy:verify
+  - yarn build
 script:
-    - PORT=8081 REACT_APP_DHIS2_BASE_URL=http://dev2.eyeseetea.com:8083 yarn start &
-    - yarn run wait-on http://localhost:8081 && sleep 10s
```

Comment:
So I guess travis add node_modules binaries to tthe path and we don't need `yarn run xyz` / `npx xyz`.